README.md

```diff
@@ -140,6 +140,10 @@ kuzzle
 });
 ```
 
+## Building manually
+
+Clone this github repository and run ``npm run build``. A ``dist`` directory will be created, containing a plain browserified version of this SDK, and a minified version.
```

Comment:
I removed this section because I've rewritten the README. There is no "browserified" version anymore, and since this step is required to use the SDK in a browser, I've already detailed the procedure in the "Installation/Javascript" section.

I think this section should go.